src/runtime-tools/linux/setup.sh

```diff
@@ -84,6 +85,17 @@ fi
 
 chmod -R a+rx /onefuzz/tools/linux
 
+if type apt > /dev/null 2> /dev/null; then
+    sudo apt update
+    sudo apt install -y gdb gdbserver
```

Comment:
Does this mean we're now installing `gdbserver` on all `apt` VMs?

Reply:
Yes.  We'll need to do this anyways once we move to performing live-repro in managed scalesets.